sys/fido2/ctap/Kconfig

```diff
@@ -108,12 +110,19 @@ config FIDO2_CTAP_UP_BUTTON_FLANK_RISING
 
 endchoice
 
-config FIDO2_CTAP_FLASH_START_PAGE
-    int "First flash page to store data in"
-    default -1
+config FIDO2_CTAP_NUM_FLASHPAGES
+    int "Amount of flashpages to use"
+    range 2 256
+    default 4
     help
-        Configuring this incorrectly can lead to firmware corruption so make sure
-        the flash page is located after the firmware.
+        Configure how many flashpages are used to store FIDO2 CTAP data.
+
+        To save a credential in flash memory, roughly 156 bytes are needed. This
```

Comment:
Try to use the same language here compared to the code. rk versus state versus credential.